install/kubernetes/helm/istio/charts/grafana/dashboards/istio-basic-dashboard.json

```
@@ -0,0 +1,1090 @@
+        },
+        {
+          "alias": "",
+          "colorMode": null,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "dateFormat": "YYYY-MM-DD HH:mm:ss",
+          "decimals": 2,
+          "mappingType": 1,
+          "pattern": "_tmp_destination_workload_var",
+          "thresholds": [],
+          "type": "hidden",
+          "unit": "short"
+        },
+        {
+          "alias": "",
+          "colorMode": null,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "dateFormat": "YYYY-MM-DD HH:mm:ss",
+          "decimals": 2,
+          "mappingType": 1,
+          "pattern": "version",
+          "thresholds": [],
+          "type": "hidden",
+          "unit": "short"
+        },
+        {
+          "alias": "",
+          "colorMode": null,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "decimals": 2,
+          "pattern": "/.*/",
+          "thresholds": [],
+          "type": "number",
+          "unit": "short"
+        }
+      ],
+      "targets": [
+        {
+          "expr": "label_join(label_join(sum(irate(envoy_http_downstream_rq[1m])) by (app, version, namespace), \"_tmp_destination_workload_var\", \".\", \"app\", \"namespace\"), \"destination_workload_var\", \"/\", \"_tmp_destination_workload_var\", \"version\")",
+          "format": "table",
+          "instant": true,
+          "interval": "",
+          "intervalFactor": 1,
+          "legendFormat": "",
+          "refId": "A"
+        },
+        {
+          "expr": "histogram_quantile(0.50, sum(rate(envoy_http_downstream_rq_time_bucket[1m])) by (le, app, version, namespace))",
+          "format": "table",
+          "instant": true,
+          "intervalFactor": 1,
+          "legendFormat": "",
+          "refId": "B"
+        },
+        {
+          "expr": "histogram_quantile(0.90, sum(rate(envoy_http_downstream_rq_time_bucket[1m])) by (le, app, version, namespace))",
+          "format": "table",
+          "instant": true,
+          "intervalFactor": 1,
+          "legendFormat": "",
+          "refId": "C"
+        },
+        {
+          "expr": "histogram_quantile(0.99, sum(rate(envoy_http_downstream_rq_time_bucket[1m])) by (le, app, version, namespace))",
+          "format": "table",
+          "instant": true,
+          "intervalFactor": 1,
+          "legendFormat": "",
+          "refId": "D"
+        },
+        {
+          "expr": "(sum(irate(envoy_http_downstream_rq[1m])) by (app, version, namespace) - sum(irate(envoy_http_downstream_rq{response_code_class=\"5xx\"}[1m])) by (app, version, namespace))/sum(irate(envoy_http_downstream_rq[1m])) by (app, version, namespace)",
```

I'm getting 50% success rate with this query. I think this is due to the element with response_code_class AND the element without that tag reporting values, which are SUMed resulting in the total requests being 2x what they really are. I changed this query to this and it works as expected, but would appreciate someone reviewing:

```suggestion
          "expr": "(sum(irate(envoy_http_downstream_rq[1m])) by (app, version, namespace) - sum(irate(envoy_http_downstream_rq{response_code_class=\"5xx\"}[1m])) by (app, version, namespace))/sum(irate(envoy_http_downstream_rq[1m])) by (app, version, namespace, response_code_class)",
```